tests/parsers/appbio_quantstudio_designandanalysis/testdata/appbio_quantstudio_designandanalysis_QS7Pro_PCR_with_Melt_example09.json

```diff
@@ -501,7 +502,7 @@
                                 }
                             },
                             "melting curve data cube": {
-                                "label": "melt curve",
+                                "label": "melting curve",
```

Comment:
NOTE to reviewer: this and the unit change in were bugs in the existing implementation